tasks/ghost-os.yml

```diff
@@ -4,3 +4,9 @@
   template: src=etc/init/ghost.conf.j2 dest=/etc/init/ghost.conf
   notify: restart ghost
   tags: [ghost]
```

Comment:
Here It will be nice to have a condition to install It only if It's for versions < than 15.04